changelogs/fragments/int-test-fix.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - Integration test tag fix
```

Comment:
I really don't like this fragment. If I found this in a changelog with no context, I would not feel any more enlightened than when I started, which seems to indicate that this fragment only exists to appease zuul and not to help anyone.

A more detailed change like "Changed integration tests to only wait for netconf availability when netconf is being tested" with a link back to the PR would be fine. A new changelog fragment type like `internal_changes` or `testing_changes` might be better. Skipping changelog fragment check on non-coding changes is my personal preference, but could be going too far.